engine/swarm/stack-deploy.md

```diff
@@ -0,0 +1,250 @@
+---
+description: How to deploy a stack to a swarm
+keywords:
+- guide, swarm mode, composefile, stack, compose, deploy
+title: Deploy a stack to a swarm
+---
+
+When running Docker Engine in swarm mode, you can use `docker stack deploy`
+(or just `docker deploy`) to deploy a complete application stack to the swarm.
```

Comment:
We're planning on keeping `docker deploy` experimental in 1.13 so that we can change it later if we need to.  We might want to avoid calling it out here since it won't be available to everyone like `docker stack deploy`.